README.rst

```diff
@@ -1,67 +1,84 @@
+.. _CircleCI: https://circleci.com/gh/scikit-learn-contrib/project-template/tree/master
+
+project-template - A template for scikit-learn contributions
+============================================================
 
 **project-template** is a template project for
-[scikit-learn](http://scikit-learn.org/)
-compatible extensions.
+[scikit-learn](http://scikit-learn.org/) compatible extensions.
 
 It aids development of estimators that can be used in scikit-learn pipelines
 and (hyper)parameter search, while facilitating testing (including some API
 compliance), documentation, open source development, packaging, and continuous
 integration.
 
-## Important Links
-HTML Documentation - http://contrib.scikit-learn.org/project-template/
+Important Links
+---------------
+
+HTML Documentation_
+
+.. _documentation: http://contrib.scikit-learn.org/project-template/
+
+Installation and Usage
+----------------------
 
-## Installation and Usage
 The package by itself comes with a single module and an estimator. Before
 installing the module you will need `numpy` and `scipy`.
-To install the module execute:
-```shell
-$ python setup.py install
-```
-or
-```
-pip install sklearn-template
-```
+To install the module execute::
+
+  pip install sklearn-template
 
 If the installation is successful, and `scikit-learn` is correctly installed,
-you should be able to execute the following in Python:
-```python
->>> from skltemplate import TemplateEstimator
->>> estimator = TemplateEstimator()
->>> estimator.fit(np.arange(10).reshape(10, 1), np.arange(10))
-```
+you should be able to execute the following in Python::
+
+  >>> from skltemplate import TemplateEstimator
+  >>> estimator = TemplateEstimator()
+  >>> estimator.fit(np.arange(10).reshape(10, 1), np.arange(10))
 
 `TemplateEstimator` by itself does nothing useful, but it serves as an example
 of how other Estimators should be written. It also comes with its own unit
 tests under `skltemplate/tests` which can be run using `py.test`.
 
-## Creating your own library
+Creating your own library
+-------------------------
+
+1. Cloning
+~~~~~~~~~~
+
+Clone the project into your computer by executing::
+
+  $ git clone https://github.com/scikit-learn-contrib/project-template.git
 
-### 1. Cloning
-Clone the project into your computer by executing
-```shell
-$ git clone https://github.com/scikit-learn-contrib/project-template.git
-```
 You should rename the `project-template` folder to the name of your project.
 To host the project on Github, visit https://github.com/new and create a new
-repository. To upload your project on Github execute
-```shell
-$ git remote set-url origin https://github.com/username/project-name.git
-$ git push origin master
-```
+repository. To upload your project on Github execute::
+
+  $ git remote set-url origin https://github.com/username/project-name.git
+  $ git push origin master
```

Comment:
I personally find it safer to have origin pointing at the fork and upstream at the official repo.